cmd/awscollector/Dockerfile

```diff
@@ -72,6 +76,7 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 COPY --from=package /workspace/awscollector /awscollector
 COPY --from=package /workspace/config/ /etc/
 COPY --from=package /workspace/healthcheck /healthcheck
+COPY --from=ecstarget ecs_sd_targets.yaml /etc/
```

Comment:
this same workaround could be implemented directly in the ecsobserver extension with the advantage that it would be future proof. This workaround is fragile and will only work if customers use this exact location for the sd_targets file.

Please consider fixing this upstream by creating an empty file based on the configuration.
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/observer/ecsobserver/extension.go#L42